CondaPkg.toml

```diff
@@ -0,0 +1,9 @@
+channels = ["anaconda", "conda-forge"]
+
+[deps]
+# Conda package names and versions
+python = ">=3.7,<4"
+# CondaPkg uses mamba by default
+scipy =  ">=1.9"
```

Comment:
Do we still need this?

Reply:
Oh, looks like we have a separate CondaPkg.toml for test. I'll remove, test, and then merge if it's all good.